core/src/test/java/org/apache/iceberg/TestIncrementalDataTableScan.java

```diff
@@ -60,7 +60,7 @@ public void testInvalidScans() {
     AssertHelpers.assertThrows(
         "from and to snapshots cannot be the same, since from snapshot is exclusive and not part of the scan",
         IllegalArgumentException.class, "from and to snapshot ids cannot be the same",
-        () -> appendsBetweenScan(1, 1));
+        () -> appendsBetweenScan(1L, 1L));
```

Comment:
need to change the `fromSnapshotId` from `1` to `1L` because the type is `Long` now. there is no auto type conversion from `int` to `Long`.

Since I am changing the constants for `fromSnapshotId`, I also changed the `toSnapshotId` as well to be consistent.